### PR TITLE
libvirt: concept, 1st draft

### PR DESCRIPTION
**Pull request by tbazant**
rendered PDF is attached
[concept-manage-virtual-machines-libvirt_color_en.pdf](https://github.com/SUSE/doc-modular/files/7076114/concept-manage-virtual-machines-libvirt_color_en.pdf)

### CODE REVIEW

File: xml/concept-manage-virtual-machines-libvirt.xml [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+  <para>
+   The &libvirtd; service runs on the server and performs management tasks for
+   virtualized guests, for example starting, stopping, and migrating guests
+   between host servers. The &libvirt; client libraries and utilities connect
+   to &libvirtd; to issue tasks and collect information about the configuration
+   and resources of the host system and guests.
+  </para>
+  <para>
+   The configuration of each virtual machine is stored in an XML file. You can
+   manage their configuration in the following ways:
```
Comment (cjschroder) [inline xml/concept-manage-virtual-machines-libvirt.xml:93]: configuration > configurations

File: xml/concept-manage-virtual-machines-libvirt.xml [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+   </listitem>
+  </itemizedlist>
+ </section>
+ <section xml:id="what-is-libvirt">
+  <!--to create unique section IDs, amend the TOPIC-ID (without prefix)-->
+  <!--example: what-is-ntp-time-synchronization-->
+  <title>What is &libvirt;?</title>
+  <!-- based on https://susedoc.github.io/doc-sle/main/single-html/SLES-virtualization/#cha-tools-intro -->
+  <para>
+   &libvirt; is a collection of software that provides a common API for
```
Comment (cjschroder) [inline xml/concept-manage-virtual-machines-libvirt.xml:68]: Spell out API on its first use

File: xml/concept-manage-virtual-machines-libvirt.xml [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+   </listitem>
+   <listitem>
+    <para>
+     Ability to manage remote virtual machines.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     On a host running the &libvirtd; service, you can manage various types of
+     storage that is available to &vmguest;s.
```
Comment (cjschroder) [inline xml/concept-manage-virtual-machines-libvirt.xml:141]: is available > are available

**Review by cjschroder — CHANGES_REQUESTED** [review]
Just a few small grammar corrections.

File: xml/concept-manage-virtual-machines-libvirt.xml (hunk 2 above)
Comment (tbazant) [reply xml/concept-manage-virtual-machines-libvirt.xml:68]:
```suggestion
   &libvirt; is a collection of software that provides a common API (Application Programming Interface) for
```

File: xml/concept-manage-virtual-machines-libvirt.xml (hunk 1 above)
Comment (tbazant) [reply xml/concept-manage-virtual-machines-libvirt.xml:93]:
```suggestion
   manage their configurations in the following ways:
```

File: xml/concept-manage-virtual-machines-libvirt.xml (hunk 3 above)
Comment (tbazant) [reply xml/concept-manage-virtual-machines-libvirt.xml:141]:
```suggestion
     storage that are available to &vmguest;s.
```

**Review by tbazant — COMMENTED** [review]
included suggestions from @cjschroder

**Pull request merged**